libraries/Microsoft.Bot.Builder.Dialogs/Memory/Scopes/SettingsMemoryScope.cs

```diff
@@ -18,6 +18,15 @@ namespace Microsoft.Bot.Builder.Dialogs.Memory.Scopes
     /// </summary>
     public class SettingsMemoryScope : MemoryScope
     {
+        private static readonly List<string> _blockingList = new List<string>
+        {
+            "MicrosoftAppPassword",
```

Comment:
Missing some entries for the new runtime, listing them below. 

- app insights: `runtimeSettings.telemetry.options.connectionString`
- cosmosdb: `CosmosDbPartitionedStorage.authKey`
- blob transcript: `runtimeSettings.features.blobTranscript.connectionString`
- blob storage: `BlobsStorage.connectionString`